### PR TITLE
Adding Solr 

### PR DESCRIPTION
**Pull request by vinamartin**
Refactoring dependency hell

### CODE REVIEW

File: multi-int-store/src/main/java/com/connexta/multiintstore/controllers/StorageController.java [hunk 1]
```diff
@@ -36,21 +30,4 @@
   public StorageController(StorageManager storageManager) {
     this.manager = storageManager;
   }
-
-  @PostMapping(path = "/store/{ingestID}", consumes = "application/json")
-  public ResponseEntity store(
```
Comment (vinamartin) [inline multi-int-store/src/main/java/com/connexta/multiintstore/controllers/StorageController.java]: Oh this change shouldn't be in here, not sure why it is

File: multi-int-store/src/main/java/com/connexta/multiintstore/storage/persistence/config/SolrConfig.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+@Configuration
+@EnableSolrRepositories(
+    basePackages = "com.connexta.multiintstore.storage.persistence.repository",
+    namedQueriesLocation = "classpath:solr-named-queries.properties")
+@ComponentScan
+public class SolrConfig {
+
+  @Bean
+  public SolrClient solrClient() {
+    return new HttpSolrClient.Builder("http://localhost:8983/solr").build();
```
Comment (vinamartin) [inline multi-int-store/src/main/java/com/connexta/multiintstore/storage/persistence/config/SolrConfig.java]: This should be refactored out to use properties once the cassandra PR is merged

File: multi-int-store/build.gradle [hunk 3]
```diff
@@ -4,6 +4,7 @@ plugins {
 }
 
 dependencies {
+	implementation "org.springframework.data:spring-data-solr:4.0.8.RELEASE"
```
Comment (Kjames5269) [inline multi-int-store/build.gradle]: Out of curiosity why are we using this version of the spring framework?

Comment (vinamartin) [reply multi-int-store/build.gradle]: Newest version

**Comment by emmberk** [comment]
build now

**Pull request merged**